### PR TITLE
Smarthub: Update to Attekmi Endpoint

### PR DESCRIPTION
**Pull request by SmartHubSolutions**
We change adapter name from Smarthub to Attekmi,
changing endpoints but keep biddercode 'smarthub'

Docs PR https://github.com/prebid/prebid.github.io/pull/5699

### CODE REVIEW

**Comment by bretg** [comment]
We discussed this in committee. Renaming an adapter is covered in the FAQ -- https://docs.prebid.org/faq/prebid-server-faq.html#how-can-we-rename-our-bid-adapter

Please add attekmi as an alias of smarthub. It's confusing for publishers to navigate re-brandings without a transition period where both old and new names are available.

**Comment by SmartHubSolutions** [comment]
Attekmi cannot work as an alias, does that mean we have to create a completely new adapter as Smarthub? 
In this case, we will be forced to change all aliases associated with Smarthub

**Comment by bretg** [comment]
> Attekmi cannot work as an alias

It certainly could if you wanted it to. 

You started this thread with "We change adapter name from Smarthub to Attekmi". Maybe we should step back and confirm what you're actually doing here. This PR just changes the URLs and the titles, but doesn't actually attempt to get rid of the 'smarthub' biddercode.

Let's make sure we're on the same page about what's important, which is that all of your publisher customers have coded their pages/stored requests with the 'smarthub' biddercode. They've also set up build processes that include and enable the 'smarthub' adapter. What Prebid is picky about is forcing publishers to change either their pages or their build processes outside of a major release. That's called a 'breaking change'.

So question - are you going to be asking your customers to change their pages to replace 'smarthub' with 'attekmi'? That's most rebranders want to do, which is why we ask for aliases. This way, you can start transitioning publishers to the new brand but nothing breaks for those who are too busy to change.

If you're just changing the technical DNS endpoints, then there would be no need to change the branding in some of these files. The docs PR covering this suggests that attekmi is intended to be used a biddercode.

Some rebranders don't worry about changing the biddercodes. For instance, the companies 'rubicon' and 'appnexus' haven't existed for many years, but that's still what the bid adapters are called. The fact that 'Magnite' and 'Microsoft' own these biddercodes is mentioned in the documentation, but there's no separate docs files for them.

**Comment by SmartHubSolutions** [comment]
Thanks for the detailed answer!

In this PR we only change the URLs and titles and leave the 'smarthub' biddercode.

Our clients use and will continue to use the 'smarthub' biddercode.

We are not going to ask our clients to change the current biddercode.

We are aware of the experience of other partners, thank you for the clarification.

File: adapters/smarthub/smarthub_test.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ import (
 
 func TestJsonSamples(t *testing.T) {
 	bidder, buildErr := Builder(openrtb_ext.BidderSmartHub, config.Adapter{
-		Endpoint: "http://prebid.smart-hub.io/pbserver?partnerName={{.Host}}&seat={{.AccountID}}&token={{.SourceId}}"}, config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"})
+		Endpoint: "https://prebid.attekmi.com/pbserver?partnerName={{.Host}}&seat={{.AccountID}}&token={{.SourceId}}"}, config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"})
```
Comment (bsardo) [inline adapters/smarthub/smarthub_test.go:14]: I recommend using a fake endpoint in the tests for maintenance reasons so you don't have to change all of these files if the endpoint ever changes again.

File: static/bidder-info/smarthub.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
-endpoint: "http://prebid.smart-hub.io/pbserver?partnerName={{.Host}}&seat={{.AccountID}}&token={{.SourceId}}"
+endpoint: "https://prebid.attekmi.com/pbserver?partnerName={{.Host}}&seat={{.AccountID}}&token={{.SourceId}}"
 maintainer:
-  email: "support@smart-hub.io"
+  email: "prebid@attekmi.com"
```
Comment (bsardo) [inline static/bidder-info/smarthub.yaml:3]: We sent a test email to this address. Please respond back with "Confirmed".

File: static/bidder-info/smarthub.yaml [hunk 3]
```diff
@@ -13,7 +13,7 @@ capabilities:
       - video
       - native
 userSync:
-  # smarthub supports user syncing, but requires configuration by the host. contact this
+  # Attekmi supports user syncing, but requires configuration by the host. contact this
```
Comment (bsardo) [inline static/bidder-info/smarthub.yaml:16]: Nitpick: perhaps change this to "Attekmi's Smarthub supports..."

File: static/bidder-params/smarthub.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-  "title": "SmartHub Adapter Params",
-  "description": "A schema which validates params accepted by the SmartHub adapter",
+  "title": "Attekmi Adapter Params",
+  "description": "A schema which validates params accepted by the Attekmi adapter",
   "type": "object",
   "properties": {
     "partnerName": {
       "type": "string",
-      "description": "SmartHub unique partner name",
+      "description": "Attekmi unique partner name",
```
Comment (bsardo) [inline static/bidder-params/smarthub.json:9]: I suggest changing these three occurrences to "Attekmi Smarthub"

File: static/bidder-info/smarthub.yaml (hunk 2 above)
Comment (SmartHubSolutions) [reply static/bidder-info/smarthub.yaml:3]: Email was confirmed

File: static/bidder-params/smarthub.json (hunk 4 above)
Comment (SmartHubSolutions) [reply static/bidder-params/smarthub.json:9]: Our setup instructions only have "Attekmi partner name", without "Smarthub"

File: static/bidder-info/smarthub.yaml (hunk 3 above)
Comment (SmartHubSolutions) [reply static/bidder-info/smarthub.yaml:16]: We provide the customer with updated setup instructions, so a name other than Attekmi would be misleading to the customer.

File: adapters/smarthub/smarthub_test.go (hunk 1 above)
Comment (SmartHubSolutions) [reply adapters/smarthub/smarthub_test.go]: Endpoint changed to prebid.example.com

**Review by przemkaczmarek — APPROVED** [review]

File: static/bidder-params/smarthub.json [hunk 5]
```diff
@@ -1,12 +1,12 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "title": "SmartHub Adapter Params",
-  "description": "A schema which validates params accepted by the SmartHub adapter",
+  "title": "Attekmi Adapter Params",
```
Comment (bretg) [inline static/bidder-params/smarthub.json:3]:
```suggestion
  "title": "Attekmi (formerly SmartHub) Adapter Params",
```

File: static/bidder-params/smarthub.json [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
+  "description": "A schema which validates params accepted by the Attekmi adapter",
```
Comment (bretg) [inline static/bidder-params/smarthub.json:4]:
```suggestion
  "description": "A schema which validates params accepted by the Attekmi (formerly SmartHub) adapter",
```

File: static/bidder-params/smarthub.json (hunk 4 above)
Comment (bretg) [inline static/bidder-params/smarthub.json:9]:
```suggestion
      "description": "Attekmi (formerly SmartHub) unique partner name",
```

**Review by bretg — CHANGES_REQUESTED** [review]
I am asking you to have a transition period where people can get used to your rebranding... at least through a major release. i.e. publishers and host companies should not have to be aware of this rebranding for things to work or to find documentation. I view this as no different than appnexus/MicroSoft and rubicon/Magnite.

File: static/bidder-params/smarthub.json (hunk 6 above)
Comment (SmartHubSolutions) [reply static/bidder-params/smarthub.json]: I changed description

**Review by przemkaczmarek — APPROVED** [review]

**Review by bsardo — APPROVED** [review]

**Pull request merged**